### PR TITLE
standardize errors and logs of server pkg

### PR DESCRIPTION
**Pull request by drgarcia1986**
This change is [Reviewable](https://reviewable.io/reviews/luizalabs/teresa-api/218)

### CODE REVIEW

File: pkg/server/app/app.go [hunk 1]
```diff
@@ -88,26 +90,36 @@ func (ops *AppOperations) Create(user *storage.User, app *App) error {
 	}
 
 	if err := ops.kops.CreateNamespace(app, user.Email); err != nil {
-		return err
+		if ops.kops.IsAlreadyExists(err) {
+			return ErrAlreadyExists
+		}
+		return teresa_errors.New(teresa_errors.ErrInternalServerError, err)
```
Comment (aguerra) [inline pkg/server/app/app.go]: Maybe I would go with teresa_errors.NewInternalServerError(err)

File: pkg/server/app/app.go [hunk 2]
```diff
@@ -201,9 +212,9 @@ func (ops *AppOperations) TeamName(appName string) (string, error) {
 	teamName, err := ops.kops.NamespaceLabel(appName, TeresaTeamLabel)
 	if err != nil {
 		if ops.kops.IsNotFound(err) {
-			return "", newAppErr(ErrNotFound, err)
+			return "", teresa_errors.New(ErrNotFound, err)
```
Comment (aguerra) [inline pkg/server/app/app.go]: Maybe just return "", ErrNotFound

Comment (drgarcia1986) [inline pkg/server/app/app.go]: Yeah, your right!

File: pkg/server/app/app.go (hunk 1 above)
Comment (drgarcia1986) [inline pkg/server/app/app.go]: Agree. This little change will simplify a lot.

**Pull request merged**